overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -20572,9 +20572,9 @@
           "requiredItems:9": {
             "0:10": {
               "Count:3": 1,
-              "Damage:2": 32005,
+              "Damage:2": 0,
               "OreDict:8": "",
```

Comment:
This doesn't look right...

Reply:
Good eye--checked and it's for this quest, so I think it might be correct?

![image](https://github.com/Nomi-CEu/Nomi-CEu/assets/5544386/c9fc7af6-23a5-456e-96d4-ad4ff14b0481)

Reply:
It should be meta 32107.

Reply:
Ohh yeah that makes more sense. Fixed 👍